operators/RockX/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name”: "RockX",
```

Comment:
Hi @AlexRockX , this seems to an invalid json. the quotes after `name` is not the right one.

Reply:
@shrimalmadhur Sorry for the formatting error, the correct quotes have been updated in the latest commit,  please let us know if it can be used.